android/gradle.properties

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
These lines below here are related to a merge conflict that weren't resolved

<<<<<<< HEAD
=======
>>>>>>>